components/infobox/commons/infobox_manufacturer.lua

```diff
@@ -0,0 +1,132 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Manufacturer
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Namespace = require('Module:Namespace')
+
+local BasicInfobox = Lua.import('Module:Infobox/Basic', {requireDevIfEnabled = true})
+local Locale = Lua.import('Module:Locale', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Header = Widgets.Header
+local Title = Widgets.Title
+local Center = Widgets.Center
+local Customizable = Widgets.Customizable
+local Builder = Widgets.Builder
+
+---@class ManufacturerInfobox: BasicInfobox
+local Manufacturer = Class.new(BasicInfobox)
+
+---@param frame Frame
+---@return Html
+function Manufacturer.run(frame)
```

Comment:
imo remove this function 
imo it should be set by the /Custom

Reply:
the create infobox one? it doesnt need it?

Reply:
Ah I see!